docs/using/recipes.md

```diff
@@ -568,3 +568,20 @@ docker run -it --rm \
     -v /tmp/.X11-unix:/tmp/.X11-unix \
     jupyter/minimal-notebook
 ```
+
+## Add ijavascript kernel to container
+
+The example below is a Dockerfile to install the ijavascript kernel.
+
+```dockerfile
+# use one of the jupyter docker stacks images
+FROM jupyter/scipy-notebook:85f615d5cafa
+
+# configure where npm installs global packages & install ijavascript
+RUN npm config set prefix "${HOME}"
+RUN npm install -g ijavascript
+RUN "${HOME}"/bin/ijsinstall
```

Comment:
why is the global packages updated to be installed in the home folder like this as part of this?

If its motivated, I think a comment to clarify why is relevant to add.

Reply:
no longer installing in home.